WordPress/src/main/java/org/wordpress/android/ui/posts/PostPreviewActivity.java

```diff
@@ -284,7 +287,9 @@ private void revertPost() {
     private void publishPost() {
         if (!isFinishing() && NetworkUtils.checkConnection(this)) {
             if (!mPost.isLocalDraft()) {
-                AnalyticsUtils.trackWithSiteDetails(AnalyticsTracker.Stat.EDITOR_UPDATED_POST, mSite);
+                Map<String, Object> properties = new HashMap<>();
+                properties.put("has_gutenberg_blocks", PostUtils.contentContainsGutenbergBlocks(mPost.getContent()));
```

Comment:
Niptick: what do you think on use a string constant for  `has_gutenberg_blocks` property?

Reply:
Addressed in 9f08330d66